apps/client/src/helpers/featureFlag/featureFlag.ts

```diff
@@ -0,0 +1,16 @@
+export const Features = {
```

Comment:
Thank you for adding a feature flag to complete this feature incrementally!

Reply:
This is a great start for a feature flag module and likely adequate for our initial release.

In the near future, we should consider using a library like https://github.com/Unleash/unleash to handle this, which will give us much more control over releases. 

Additionally, it would be worth considering a feature flag plug-in we can use across iot-application, iot-app-kit, and future additions.

Reply:
Sounds good. I mainly wanted something simple to unblock the incremental delivery, but I agree we should use a library and make it shareable.